debian/control

```diff
@@ -14,23 +14,23 @@ Build-Depends:
  ca-certificates,
  cpio,
  debhelper (>= 10~),
- docbook-xml,
- docbook-xsl,
+ dh-exec,
+ docbook-xml <!nodoc>,
+ docbook-xsl <!nodoc>,
  e2fslibs-dev,
  elfutils,
  fuse,
  gjs [!sparc64],
  gobject-introspection,
- gtk-doc-tools (>= 1.15),
- libarchive-dev (>= 2.8.0),
+ gtk-doc-tools <!nodoc>,
+ libarchive-dev,
  libattr1-dev,
- libc6-dev (>= 2.19-18+deb8u4endless4) [!amd64 !i386],
  libcap-dev,
  libfuse-dev,
- libgirepository1.0-dev (>= 1.34.0),
+ libgirepository1.0-dev,
  libglib2.0-dev (>= 2.40.0),
- libgpgme11-dev (>= 1.1.8),
- liblzma-dev (>= 5.0.5),
+ libgpgme11-dev,
+ liblzma-dev,
```

Comment:
I guess it's fine to drop all the version requirements since we likely have them on master and eos3.2.